sdk/resourcemanager/Azure.ResourceManager/src/ArmOperation.cs

```diff
@@ -3,13 +3,34 @@
 
 // <auto-generated/>
 
-#nullable disable
+#nullable enable
 
+using Azure.Core;
+using System.ClientModel.Primitives;
 
 namespace Azure.ResourceManager
 {
     /// <inheritdoc/>
     public abstract class ArmOperation : Operation
     {
+        /// <summary>
+        /// Rehydrates an operation from a <see cref="RehydrationToken"/>.
+        /// </summary>
+        /// <param name="client">The Arm client.</param>
+        /// <param name="rehydrationToken">The rehydration token.</param>
+        /// <param name="options">The Arm client options.</param>
+        /// <returns>The long-running operation.</returns>
+        public static Operation Rehydrate(ArmClient client, RehydrationToken? rehydrationToken, ArmClientOptions? options = null)
```

Comment:
We should return ArmOperation here

Reply:
Makes sense, we should return ArmOperation for Arm usage.